packages/theme/package.json

```
@@ -32,7 +32,15 @@
     "test": "backstage-cli package test"
   },
   "dependencies": {
-    "@material-ui/core": "^4.12.2"
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@material-ui/core": "^4.12.2",
+    "@mui/material": "^5.11.2",
+    "@mui/styles": "^5.11.2"
+  },
+  "peerDependencies": {
+    "@types/react": "^16.13.1 || ^17.0.0",
+    "react": "^16.13.1 || ^17.0.0"
```

I suspect it needs to be:

```suggestion
    "@types/react": "^17.0.0",
    "react": "^17.0.0"
```

as `@mui/material` excludes React 16.13.1. If it's an issue, than it relates to https://github.com/mui/material-ui/pull/35232#discussion_r1030906426.

---

Ah alright, thank you for the heads up! 👍 

Not sure where we end up with this to be honest, might be that it just ends up being a documented constraint if this package should be possible to use with only MUI v4. On the other hand we might bring in MUI v5 as a required dependency here pretty soon and then we of course need to align versions. I do agree with people in the thread over in `mui/` that it's a bit unfortunate to drop React >=16.8 support.